### PR TITLE
Documentation for CHT Application tests

### PR DESCRIPTION
**Pull request by pkitutu**

### CODE REVIEW

File: content/en/apps/guides/application-tests/features_to_test.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: "Common features of the app that require automated testing"
+linkTitle: "Features to test"
+weight: 2
```
Comment (kennsippell) [inline content/en/apps/guides/application-tests/features_to_test.md:4]: i'm a bit unclear on the difference between this content vs [this existing content](http://docs.communityhealthtoolkit.org/cht-conf-test-harness/). I'm wondering if they can somehow be effectively combined?  I think the location of this content is great, and there are some additions here that seem very valuable - but having deep links into the harness documentation would be very valuable. For example `getContactSummary` should link to [this](http://docs.communityhealthtoolkit.org/cht-conf-test-harness/Harness.html#getContactSummary).

Comment (pkitutu) [reply content/en/apps/guides/application-tests/features_to_test.md:4]: Thanks for the feedback @kennsippell. I think we can everything incorporated in the new documentation or we could just update the old documentation. Will make the suggested deep links

**Comment by kennsippell** [comment]
@pkitutu  Confirming that this is still on your radar?

**Comment by pkitutu** [comment]
[Issue #32](https://github.com/medic/app-services-team/issues/32) taken on by @binokaryg

**Comment by binokaryg** [comment]
New PR: #698

**Pull request closed without merging**